src/Compatibility/Microsoft.DotNet.ApiCompatibility/Microsoft.DotNet.ApiCompatibility.csproj

```diff
@@ -12,6 +11,7 @@
     <!-- We pin the versions of the assets that ship with the SDK in order to be able to run on older SDKs. 
     Currently we support .NET 6 Preview5 or later. -->
     <PackageReference Include="Microsoft.CodeAnalysis" Version="4.0.0-1.21304.5" />
+    <PackageReference Include="Microsoft.Build.Framework" Version="$(MicrosoftBuildFrameworkVersion)" ExcludeAssets="Runtime" />
```

Comment:
Why do we have the `ExcludeAssets`? Is it so that we don't carry roslyn with us? If so could you add a comment?